project.md

```diff
@@ -40,7 +67,12 @@ We are inspired by lightweight practices and process to keep innovation flowing;
     * *Closed* [Issues]({{site.repo_url}}/issues) which did not come to fruition
     * *Closed* [Pull Requests]({{site.repo_url}}/pulls) which did not come to fruition
 * Cards are collectively created and refined inspired by [User Story](https://en.wikipedia.org/wiki/User_story) form
-* We are inspired by [Empirical Principles](https://www.scrum.org/resources/blog/three-pillars-empiricism-scrum) collaborating continuously as it makes sense
+* We are inspired by [Empirical Principles](https://www.scrum.org/resources/blog/three-pillars-empiricism-scrum) when collaborating
+* We collaborate continuously in a weekly hour long Mat session in this following form:
```

Comment:
Lines 70 and 71 should be combined ...

We are inspired by Empirical Principles performing a weekly inspection in a hour long mat session: